README.md

```diff
@@ -1 +1,65 @@
-# Project-SpaceX
\ No newline at end of file
+# React and Redux
+
+If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
+This GitHub Action is going to run [ESLint](https://eslint.org/) and [Stylelint](https://stylelint.io/) to help you find style issues.
+
+[Stylelint](https://stylelint.io/) is a linter for your stylesheets that helps you avoid errors and enforce conventions.
```

Comment:
Bro This is not the Readme.md file you will work on. The previous Readme.md file (Project-spaceX) was perfect.